_posts/EternalFootman/2016-02-01-clicky.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+author: EternalFootman
+title: Kit's Clicky Garden Adventure
+---
+
+You may notice a theme. This is an early scene from Through the Looking Glass; it's the one where Alice explores the garden of talking flowers.
+Warning: In keeping with the original text, this game is nonsensical and rather insulting.
+
+<iframe src="https://trinket.io/embed/python/6e73347dfc?start=result" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+All in all, I'm pleased with it, but there are a few minor glitches.
+I acknowledge that I got a little carried away with the setup, but it's important for context. Is there a way to make the speed faster than 10? This would make it much more efficient.
+Also, is there a way to do the setup all at once and go directly into the game? If the whole picture could just appear, that would save time.
+As it is, the garden code is long and repetitive.
+```
+def roses():
+  gardener.penup()
+  gardener.goto(-120,120)
+  gardener.fill(True)
+  gardener.fillcolor("darkred")
+  gardener.circle(10)
+```
+Most of that repeats for each flower. I've not found a quicker method, as each is at a new coordinate.
```

Comment:
Can you pass the coordinates in as arguments to the function and then use them within it?